cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt

```diff
@@ -126,27 +126,18 @@ script "js_filter_resources", type: "javascript" do
   code <<-EOS
     var result = [];
     _.each(azure_instances, function(instance){
-
-      console.log("Instance Name: ", instance["name"])
-
       if (_.has(instance.tags, exclusion_tag)) {
         console.log(instance["name"], "SKIP")
       } else {
-        if (instance.osType === "Windows" && instance.imageOffer.indexOf("WindowsServer")==0 ){
+        if (instance.osType === "Windows" ){
           var resourceGroup = instance.id.split('/')[4]
           var tags = JSON.stringify(instance["tags"]);
           var ahub = "N/A"
-
-          console.log(instance["name"], "OS: Windows")
-
-          if (instance.licenseType === "Windows_Server") {
+          if (instance.licenseType === "Windows_Server" || instance.licenseType === "Windows_Client") {
```

Comment:
why are we looking for windows_client.  I didn't think this was supported by AHUB

Reply:
We are looking for Windows_Client license because Windows 10 will have this license which needs to be selected while creating VM based on Windows 10, and in this line of code is filtering out VMs already having these licenses. So we will be left with only Windows servers which are not having the Windows_Server license type. Thanks

Reply:
got it. thanks for explaining.